### PR TITLE
docs: update modal doc

### PR DESCRIPTION
**Pull request by zhangyu1818**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

完善文档参数

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |        -   |
| 🇨🇳 Chinese |     -      |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/modal/index.en-US.md](https://github.com/zhangyu1818/ant-design/blob/master/components/modal/index.en-US.md)
[View rendered components/modal/index.zh-CN.md](https://github.com/zhangyu1818/ant-design/blob/master/components/modal/index.zh-CN.md)

### CODE REVIEW

File: components/modal/index.zh-CN.md [hunk 1]
```diff
@@ -72,11 +72,16 @@ cover: https://gw.alipayobjects.com/zos/alicdn/3StSdUlSH/Modal.svg
 | className | 容器类名 | string | - |  |
 | content | 内容 | string \| ReactNode | - |  |
 | icon | 自定义图标 | ReactNode | &lt;QuestionCircle /> | 3.12.0 |
+| getContainer | 指定 Modal 挂载的 HTML 节点, false 为挂载在当前 dom | HTMLElement \| () => HTMLElement \| Selectors \| false | document.body |
```
Comment (xrkffgg) [inline components/modal/index.zh-CN.md:75]: 版本的 位置 留出来，如果有时间的话，顺便可以排下序~

Comment (zhangyu1818) [reply components/modal/index.zh-CN.md:75]: 是根据属性名排序吗？

**Comment by xrkffgg** [comment]
其实我想的 Modal.method() 也列出比 Modal 多的 API 就行，重复的就不用再列出来了。

**Comment by xrkffgg** [comment]
辛苦 完善一下~

**Comment by zhangyu1818** [comment]
有好几个Modal的参数Modal.method()没有

**Comment by xrkffgg** [comment]
那先这样吧

**Pull request merged**